--- gettext-runtime/intl/libgnuintl.in.h.orig   2023-06-17 06:58:50.000000000 -0400
+++ gettext-runtime/intl/libgnuintl.in.h        2023-07-01 12:21:22.071203089 -0400
@@ -385,7 +385,7 @@
 
 # if !((defined printf && defined _GL_STDIO_H) || defined GNULIB_overrides_printf) /* don't override gnulib */
 #  undef printf
-#  if defined __NetBSD__ || defined __BEOS__ || defined __CYGWIN__ || defined __MINGW32__
+#  if defined __NetBSD__ || defined __BEOS__ || defined __CYGWIN__ || defined __MINGW32__ || defined _MSC_VER
 /* Don't break __attribute__((format(printf,M,N))).
    This redefinition is only possible because the libc in NetBSD, Cygwin,
    mingw does not have a function __printf__.
