diff -ur a/gettext-runtime/config.h.in b/gettext-runtime/config.h.in
--- gettext-0.19.8.1/gettext-runtime/config.h.in.orig	2016-06-11 09:00:54.000000000 -0400
+++ gettext-0.19.8.1/gettext-runtime/config.h.in	2018-04-08 18:56:06.658690128 -0400
@@ -350,10 +350,12 @@
 #undef HAVE_GETUID
 
 /* Define if you have the iconv() function and it works. */
-#undef HAVE_ICONV
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV */
 
 /* Define to 1 if you have the <iconv.h> header file. */
-#undef HAVE_ICONV_H
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV_H */
 
 /* Define to 1 if the compiler supports one of the keywords 'inline',
    '__inline__', '__inline' and effectively inlines functions marked as such.
diff -ur a/gettext-tools/config.h.in b/gettext-tools/config.h.in
--- gettext-0.19.8.1/gettext-tools/config.h.in.orig	2016-06-11 09:01:08.000000000 -0400
+++ gettext-0.19.8.1/gettext-tools/config.h.in	2018-04-08 19:04:29.948733325 -0400
@@ -936,10 +936,12 @@
 #undef HAVE_GETUID
 
 /* Define if you have the iconv() function and it works. */
-#undef HAVE_ICONV
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV */
 
 /* Define to 1 if you have the <iconv.h> header file. */
-#undef HAVE_ICONV_H
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV_H */
 
 /* Define to 1 if you have the `inet_ntop' function. */
 #undef HAVE_INET_NTOP
