--- gettext-0.22/gettext-tools/config.h.in.orig	2023-07-28 11:27:02.945221690 -0400
+++ gettext-0.22/gettext-tools/config.h.in	2023-07-28 11:28:23.103950976 -0400
@@ -1450,10 +1450,12 @@
 #undef HAVE_GOOD_USELOCALE
 
 /* Define if you have the iconv() function and it works. */
-#undef HAVE_ICONV
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV */
 
 /* Define to 1 if you have the <iconv.h> header file. */
-#undef HAVE_ICONV_H
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV_H */
 
 /* Define to 1 if you have the `inet_ntop' function. */
 #undef HAVE_INET_NTOP
@@ -2315,11 +2317,13 @@
 #undef HAVE___XPG_STRERROR_R
 
 /* Define as const if the declaration of iconv() needs const. */
-#undef ICONV_CONST
+/* rom-properties: Disabled. */
+/* #undef ICONV_CONST */
 
 /* Define to a symbolic name denoting the flavor of iconv_open()
    implementation. */
-#undef ICONV_FLAVOR
+/* rom-properties: Disabled. */
+/* #undef ICONV_FLAVOR */
 
 /* Define to the value of ${prefix}, as a string. */
 #undef INSTALLPREFIX
--- gettext-0.22/gettext-runtime/config.h.in.orig	2023-07-28 11:27:15.394179647 -0400
+++ gettext-0.22/gettext-runtime/config.h.in	2023-07-28 11:28:35.397909457 -0400
@@ -533,10 +533,12 @@
 #undef HAVE_GOOD_USELOCALE
 
 /* Define if you have the iconv() function and it works. */
-#undef HAVE_ICONV
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV */
 
 /* Define to 1 if you have the <iconv.h> header file. */
-#undef HAVE_ICONV_H
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV_H */
 
 /* Define to 1 if the compiler supports one of the keywords 'inline',
    '__inline__', '__inline' and effectively inlines functions marked as such.
@@ -958,11 +960,13 @@
 #undef HAVE___INLINE
 
 /* Define as const if the declaration of iconv() needs const. */
-#undef ICONV_CONST
+/* rom-properties: Disabled. */
+/* #undef ICONV_CONST */
 
 /* Define to a symbolic name denoting the flavor of iconv_open()
    implementation. */
-#undef ICONV_FLAVOR
+/* rom-properties: Disabled. */
+/* #undef ICONV_FLAVOR */
 
 /* Define to the value of ${prefix}, as a string. */
 #undef INSTALLPREFIX
--- gettext-0.22/gettext-runtime/intl/config.h.in.orig	2023-06-17 07:53:52.000000000 -0400
+++ gettext-0.22/gettext-runtime/intl/config.h.in	2023-07-28 11:28:42.230886380 -0400
@@ -351,7 +351,8 @@
 #undef HAVE_GOOD_USELOCALE
 
 /* Define if you have the iconv() function and it works. */
-#undef HAVE_ICONV
+/* rom-properties: Disabled. */
+/* #undef HAVE_ICONV */
 
 /* Define if you have the 'intmax_t' type in <stdint.h> or <inttypes.h>. */
 #undef HAVE_INTMAX_T
@@ -715,7 +716,8 @@
 #endif
 
 /* Define as const if the declaration of iconv() needs const. */
-#undef ICONV_CONST
+/* rom-properties: Disabled. */
+/* #undef ICONV_CONST */
 
 /* Define to the value of ${prefix}, as a string. */
 #undef INSTALLPREFIX
