--- gettext-0.21/gettext-tools/woe32dll/gettextsrc-exports.c.orig	2019-12-23 06:21:12.000000000 -0500
+++ gettext-0.21/gettext-tools/woe32dll/gettextsrc-exports.c	2020-08-20 12:20:30.752088214 -0400
@@ -50,6 +50,7 @@
 VARIABLE(formatstring_python_brace)
 VARIABLE(formatstring_qt)
 VARIABLE(formatstring_qt_plural)
+VARIABLE(formatstring_ruby)
 VARIABLE(formatstring_scheme)
 VARIABLE(formatstring_sh)
 VARIABLE(formatstring_smalltalk)
